jest.config.js

```diff
@@ -17,6 +17,11 @@ module.exports = {
     '!**/__stories__/**',
     '!app/javascript/storybook-static/**/*.js',
   ],
+  transform: {
+    '\\.jsx?(\\.erb)?$': 'babel-jest',
```

Comment:
I was experimenting the possibility of using jsx.erb and still making jest tests pass

Reply:
Thanks to this https://github.com/usabilityhub/rails-erb-loader/issues/48#issuecomment-329644557 (but the tests are failing now)

Reply:
This might not be a good idea. But I'm doing just to see if it is possible

Reply:
I was trying to import jsx.erb in a js file. This might be wrong. (I probably misunderstood how this loader works)

Reply:
cc @nickytonline

Reply:
Thanks for the help @rhymes and @nickytonline

Reply:
I usually put things down in the pr when I'm solving an issue so that person reviewing the pr later might know what was I thinking when I was doing something. Please feel free to ignore the comments that I add in the draft PR and thanks for the help @rhymes.

Reply:
You can import an erb file when webpack builds the frontend bundles as it will be able to convert it to JS via the webpack erb-loader loader. For jest though, things only run through Babel which is why the tests fail when an ERB template is referenced. See https://github.com/rails/webpacker/issues/815. It looks like someone create a package for jest to handle this, https://www.npmjs.com/package/jest-erb-transformer . I'm not sure if we want to go in that direction though. I'm going to dig a bit into this.

Reply:
Using jsx.erb might result in some weird issues with respect to tooling (eslint might not be able to parse those files). There might exist some solution for that.